perf/lua/read_view_open_close.lua

```diff
@@ -0,0 +1,57 @@
+local fio = require('fio')
+local benchmark = require('benchmark')
+local clock = require('clock')
+
+local USAGE = [[
+   field_count <number, 10000>  - the number of fields in the space format
+   iters <number, 10>           - the number of iterations (open + close
+                                  read view)
```

Comment:
I guess number of iterations is not convenient parameter. We need to change it too if we change fields count so that perf test running time does not become small (which will increase noise in measurements). Let's instead add minimum running time as in `perf/lua/column_scan.lua`.